TAGradingServer/app/models/ElectronicGradeable.php

```diff
@@ -151,6 +152,12 @@ class ElectronicGradeable {
      * @var array
      */
     public $questions_count = array();
+    
+    
+    /**
+        The maximum score for autograding excluding extra credit
+    **/
```

Comment:
change this to be `/** @var float The maximum score for autograding excluding extra credit */`